### ISSUE TITLE
Getting the list of voices is really slow because of Pydantic validation

### ISSUE THREAD

**Issue by yunusdemir** [elevenlabs/elevenlabs-python#77]
## The observation
When a user is getting the list of voices, either via `voices()` in `simple.py` or `Voices.from_api()`, the load time gets longer the more voices users have in their accounts.

## The findings
We currently have 19 voices in our account and the loading time of `voices()` is about 31 seconds. After investigating, this is caused by `computed_settings` method on the `Voice` class. The method does a request per `Voice` instance to the ElevenLabs servers to get the `VoiceSettings` because there are no settings given in the get voices route via the ElevenLabs API (`"settings":null`). 

The problem lies in lines 133-136 of `api/voice.py`:
```python
@validator("settings")
def computed_settings(cls, v: VoiceSettings, values) -> VoiceSettings:
    url = f"{api_base_url_v1}/voices/{values['voice_id']}/settings"
    return v if v else VoiceSettings(**API.get(url).json())
```

It returns `VoiceSettings` if given, but since there are no settings given in the API, they are not passed and the method will get them one by one via the API, which is slow. 

## Proposed solutions
 * Make it possible to skip getting the `VoiceSettings` when getting a list of the voices. This would be sufficient for our uses.  
 → Something I'd be able to create a PR for
 * Also sent the `VoiceSettings` in the ElevenLabs API.
   → Needs your help

**Comment by tagawa** [comment]
I get the same thing (about 11 seconds). Using the API directly (Curl) to get the voices list is fine.

**Comment by flavioschneider** [comment]
Hi @yunusdemir! This has been fixed in `v0.2.25`, the settings are not fetched by default anymore. You can call `voice.fetch_settings()` on the desired voice to individually fetch settings

**Comment by yunusdemir** [comment]
Awesome, thanks for the update @flavioschneider!
